rmw_connext_dynamic_cpp/src/functions.cpp

```diff
@@ -1786,7 +1789,7 @@ rmw_create_client(
   }
 
   const rosidl_typesupport_introspection_cpp::ServiceMembers * service_members =
-    (rosidl_typesupport_introspection_cpp::ServiceMembers *)type_support->data;
+    (const rosidl_typesupport_introspection_cpp::ServiceMembers *)type_support->data;
```

Comment:
Good point. That's another advantage for the C++ style casts: you can actually search for them reliable. I found a few more like this and converted them in 21887a32ba027efb5e52dea6ec346a110d4e6312.